src/Interpreters/Cache/LRUFileCachePriority.cpp

```diff
@@ -138,28 +147,26 @@ void LRUFileCachePriority::iterate(IterateFunc && func, const CacheGuard::Lock &
     }
 }
 
-LRUFileCachePriority::Iterator LRUFileCachePriority::LRUFileCacheIterator::remove(const CacheGuard::Lock &)
+LRUFileCachePriority::Iterator
+LRUFileCachePriority::LRUFileCacheIterator::remove(const CacheGuard::Lock &)
 {
-    return std::make_shared<LRUFileCacheIterator>(cache_priority, cache_priority->remove(queue_iter));
+    return std::make_shared<LRUFileCacheIterator>(
+        cache_priority, cache_priority->remove(queue_iter));
 }
 
 void LRUFileCachePriority::LRUFileCacheIterator::annul()
```

Comment:
what is annul?

Reply:
<img width="742" alt="Screenshot 2023-05-09 at 19 35 15" src="https://github.com/ClickHouse/ClickHouse/assets/54203879/c8b8e401-601d-4346-a73e-e6a6334548a2">